spec/fixtures/ocr/expected_bb222cc3333_00_0001.xml

```diff
@@ -26,7 +17,7 @@
 <ParagraphStyle ID="StyleId-44B0F6CE-B77C-4378-B4F7-1CCA36C002B3-" ALIGN="Left" LEFT="0." RIGHT="0." FIRSTLINE="0."/>
 <ParagraphStyle ID="StyleId-D40750C1-ED42-4B5E-A658-06B58631CB5D-" ALIGN="Left" LEFT="0." RIGHT="0." FIRSTLINE="0."/>
 </Styles>
-  <Page ID="Page1" PHYSICAL_IMG_NR="1" HEIGHT="10698" WIDTH="6927">
+<Page xmlns="http://www.loc.gov/standards/alto/ns-v3#" ID="Page1" PHYSICAL_IMG_NR="1" HEIGHT="10698" WIDTH="6927">
```

Comment:
the only real change to the structure of the XML (other than some carriage returns) is the addition of this `xmlns="http://www.loc.gov/standards/alto/ns-v3#"` to the nodes we add in (`Page`, `OcrProcessing` and `Styles`).   I couldn't find a simple way to ditch and I'm hoping it won't matter

Reply:
Seems to be fine, see https://sul-purl-stage.stanford.edu/zd178rm8212

Reply:
Nice! The default namespace for the document is `http://www.loc.gov/standards/alto/ns-v3#"` so it doesn't need to be repeated.